### DISCUSSION TITLE
Segmentation fault when running on C++

### DISCUSSION THREAD

**Discussion by liuyidesktop** [loco-3d/crocoddyl#1268] [General]
When I compile a C++ file with the Crocoddyl project, I place it in the crocoddyl/benchmark/ directory and modify the CMakeLists.txt file. This approach works well. However, when I link the libcrocoddyl.so file in other projects, the same program segfaults. There are no errors during the compilation process of Crocoddyl, and I can pass make test. Is there something wrong with my compilation method?

These are some changes I made to the CMakeLists.txt file during the compilation process:
option(BUILD_PYTHON_INTERFACE "Build the python binding" OFF)
option(BUILD_WITH_MULTITHREADS "Build the library with the Multithreading support (required OpenMP)" ON)

**Comment by cmastalli** [comment]
Hi @liuxiaoyiyixiaoliu,

Unfortunately, we're a small team that cannot afford to help people fix their code. I am sorry to tell you this, but this is in the best interests to offer future features to the community as soon as possible.

I hope you can understand why I have to close your issue. If you're willing to help us build new features, please let's discuss this with me in private. Depending on its importance we could consider whether to allocate our resources.
